lib/plugins/abstract/auth.py

```diff
@@ -121,6 +130,12 @@ def logout_hook(self, plugin_api: 'PluginApi'):
         """
         pass
 
+    @property
+    def lowercase_corpora_names(self) -> bool:
```

Comment:
This is only used in lib/controller/kontext.py to access configuration. Not a great pattern, I guess?